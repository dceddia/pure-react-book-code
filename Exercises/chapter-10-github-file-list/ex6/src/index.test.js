import ReactDOM from 'react-dom';
jest.mock('react-dom', () => ({ render: jest.fn() }));
global.document.querySelector = () => {};

import { PinterestBoard, cards } from './index';
import React from 'react';
import renderer from 'react-test-renderer';

it('renders', () => {
  const tree = renderer.create(<PinterestBoard cards={cards} />).toJSON();
  expect(tree).toMatchSnapshot();
});
