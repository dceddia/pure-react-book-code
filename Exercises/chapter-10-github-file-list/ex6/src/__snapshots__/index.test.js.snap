// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<ul
  className="board"
>
  <li>
    <div
      className="card"
    >
      <div
        className="images"
      >
        <img
          alt="AVIATION"
          className="main"
          src="787.jpg"
        />
        <img
          alt="preview 1"
          className="sub1"
          src="sub1.jpg"
        />
        <img
          alt="preview 2"
          className="sub2"
          src="sub2.jpg"
        />
      </div>
      <div
        className="summary"
      >
        <h3
          className="title"
        >
          AVIATION
        </h3>
        <div
          className="pin-count"
        >
          <strong>
            641
          </strong>
           Pins
        </div>
      </div>
      <button
        className="follow-btn"
      >
        Follow
      </button>
    </div>
  </li>
  <li>
    <div
      className="card"
    >
      <div
        className="images"
      >
        <img
          alt="Exceptional Aviation"
          className="main"
          src="787.jpg"
        />
        <img
          alt="preview 1"
          className="sub1"
          src="sub1.jpg"
        />
        <img
          alt="preview 2"
          className="sub2"
          src="sub2.jpg"
        />
      </div>
      <div
        className="summary"
      >
        <h3
          className="title"
        >
          Exceptional Aviation
        </h3>
        <div
          className="pin-count"
        >
          <strong>
            209
          </strong>
           Pins
        </div>
      </div>
      <button
        className="follow-btn"
      >
        Follow
      </button>
    </div>
  </li>
  <li>
    <div
      className="card"
    >
      <div
        className="images"
      >
        <img
          alt="Aerospace"
          className="main"
          src="787.jpg"
        />
        <img
          alt="preview 1"
          className="sub1"
          src="sub1.jpg"
        />
        <img
          alt="preview 2"
          className="sub2"
          src="sub2.jpg"
        />
      </div>
      <div
        className="summary"
      >
        <h3
          className="title"
        >
          Aerospace
        </h3>
        <div
          className="pin-count"
        >
          <strong>
            1545
          </strong>
           Pins
        </div>
      </div>
      <button
        className="follow-btn"
      >
        Follow
      </button>
    </div>
  </li>
</ul>
`;
